### ISSUE TITLE
Hackypi getting started 

### ISSUE THREAD

**Issue by Rudeboyduke** [sbcshop/HackyPi-Software#12]

**Comment by sbcshop3** [comment]
Could you please share what description you need?

**Comment by jslawson865** [comment]
Thanks Sbcshop/Hackypi-Software for reaching out. I suppose it isn’t
necessary, but I was looking into initial setup options for the product
which didn’t require Thonny, but It wasn’t an entirely problematic thing,
and was a reasonable choice of ide for the software. There were some
compatibility issues with Thonny which were atypical and were unlikely to
impact most users, but even this didn’t prevent enjoyment of the product.

This message is for the intended recipient. If you are not the specific
person this message is for, do not open or share. The information inside is
private and may be considered confidential in the state of Tennessee.
Violations of confidentiality laws will be prosecuted.

**Comment by sbcshop3** [comment]
Yeah, we chose to provide instructions based on Thonny due to its ease of use and the availability of micropython libraries on essential hardware. Of course, you can use alternative IDEs, such as VS Code and the Arduino IDE.

**Comment by jslawson865** [comment]
I see, at the time of purchase, unfortunately I was experiencing some side
effects from what seems to be a security misconfiguration which was
impacting my station as well as my ISP, that is still being researched but
it did however impair my ability to effectively utilize funny as I noticed
unusual behavior which was associated at the time with packages that we're
installed alongside the IDE which may have been compromised due to the
security troubles. I appreciate you reaching out and at some time as I'm
capable of arranging the freedom to do so I hope to put more use into the
product and perhaps provide some fun use case demonstrations for youtube.
When this is possible I would be glad to forward those your way and if
these are some thing which you find might be positive promotion of the
brand or product, I would be very happy to link to your product site and
documentation. On the topic of documentation, my purchase was made through
Amazon and I am unsure whether or not the product traditionally ships with
any form of documentation or instructions by default but it might be noted
that I didn't receive any with the device nor any packaging. It came in a
shrink wrap with a zip container. I looked about and found the
documentation necessary and likely would have been more successful than the
initial setup had it not been for a bit of preexisting DNS cache poisoning
seemingly allowing a bit of traversal in my local network and thus getting
between me and information. In the event that there is or was any official
documentation which generally comes with the device, I would be more than
happy to pay for shipping and mail a self-addressed envelope if the option
is available, however in the time since purchase I've managed to gain a
little Wan access here in the air and the paperwork would just be for my
collection so completely and totally okay if you folks are doing your part
to save on paper.
